src/notifications/util.js

```diff
@@ -1,23 +1,19 @@
-async function tryCallbackOrPromise(promise, callback = null) {
+function tryCallbackOrReturnResult(data, callback = null) {
```

Comment:
Good point, I changed it because where this code is used we receive a value, not a promise (we receive the result of await which is a value), check this line: https://github.com/bixlabs/bix-notifications/pull/22/files/018ad61d2cec50c7b999254a6c131b4ecabf51c5#diff-70c31213962f60bf3c130d530995933eR23